server/routes/README.md

```diff
@@ -0,0 +1,318 @@
+# API Routes
+An exhaustive guide to all the API routes and what to send/expect from them.  
+Path parameters are denoted by curly braces `{ }`.  
+Query parameters are denoted by angle brackets `< >`.  
+Body parameters should be passed in the body of the request, as form data.
+
+**A user must be logged in to use any of these routes**, except for user create.  
+Anytime the current user is referenced, we mean the user that is logged in.
+
+# To Do
+
+* Remove password from user objects
+* Lock down what time format to use in query and body parameters
+* Who can add people to groups? admin only or any group member? configurable?
```

Comment:
Re: adding people to groups, if the group is public, can anybody add themselves?

Reply:
@annelau99